app-forensics/yara/yara-4.3.2.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="The pattern matching swiss knife"
+DESCRIPTION="YARA is a tool aimed at (but not limited to) helping malware researchers to identify \
+and classify malware samples. With YARA you can create descriptions of malware families (or \
+whatever you want to describe) based on textual or binary patterns. "
+HOMEPAGE="https://virustotal.github.io/yara/"
+COPYRIGHT="2008-2023 VirusTotal"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/VirusTotal/yara/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a9587a813dc00ac8cdcfd6646d7f1c172f730cda8046ce849dfea7d3f6600b15"
+SOURCE_DIR="yara-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	yara$secondaryArchSuffix = $portVersion
+	cmd:yara = $portVersion
+	cmd:yarac = $portVersion
+	lib:libyara$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	yara${secondaryArchSuffix}_devel = $portVersion
+	devel:libyara$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	yara$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:automake
+	cmd:autoreconf
+	cmd:flex
+	cmd:gcc$secondaryArchSuffix
+	cmd:yacc
+	cmd:libtool
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	./bootstrap.sh
+	runConfigure --omit-dirs "binDir" ./configure --bindir=$prefix/bin
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	rm $libDir/libyara.la $libDir/libyara.a
+	prepareInstalledDevelLibs libyara
```

Comment:
Checked if using --disable-static disables building the static library?
Only 1 library involved, so:
```diff
-	prepareInstalledDevelLibs libyara
+	prepareInstalledDevelLib libyara
```